_includes/releases/v23.1/v23.1.0-alpha.5.md

```diff
@@ -0,0 +1,166 @@
+- Nodes are now considered suspect when re-joining a cluster and cannot accept lease transfers for one `server.time_after_store_suspect` window, which defaults to 30 seconds. [#97532][#97532]
+
+<h3 id="v23-1-0-alpha-5-command-line-changes">Command-line changes</h3>
+
+- The SQL shell (`cockroach sql`, `demo`) now supports the client-side commands `\l`, `\dn`, `\d`, `\di`, `\dm`, `\ds`, `\dt`, `\dv`, `\dC`, `\dT`, `\dd`, `\dg`, `\du` and `\dd` in a similar manner to PostgreSQL, including the modifier flags `S` and `+`, for convenience for users migrating from PostgreSQL. A notable difference is that when a pattern argument is specified, it should use the SQL `LIKE` syntax (with `%` representing the wildcard character) instead of PostgreSQL's glob-like syntax (with `*` representing wildcards). [#88061][#88061]
+
+<h3 id="v23-1-0-alpha-5-db-console-changes">DB Console changes</h3>
+
+- The following new metrics track memory usage of prepared statements in sessions:
+  - `sql.mem.internal.session.prepared.current`
+  - `sql.mem.internal.session.prepared.max-avg`
+  - `sql.mem.internal.session.prepared.max-count`
+  - `sql.mem.internal.session.prepared.max-max`
+  - `sql.mem.internal.session.prepared.max-p50`
+  - `sql.mem.internal.session.prepared.max-p75`
+  - `sql.mem.internal.session.prepared.max-p90`
+  - `sql.mem.internal.session.prepared.max-p99`
+  - `sql.mem.internal.session.prepared.max-p99.9`
+  - `sql.mem.internal.session.prepared.max-p99.99`
+  - `sql.mem.internal.session.prepared.max-p99.999`
+  - `sql.mem.sql.session.prepared.current`
+  - `sql.mem.sql.session.prepared.max-avg`
+  - `sql.mem.sql.session.prepared.max-count`
+  - `sql.mem.sql.session.prepared.max-max`
+  - `sql.mem.sql.session.prepared.max-p50`
+  - `sql.mem.sql.session.prepared.max-p75`
+  - `sql.mem.sql.session.prepared.max-p90`
+  - `sql.mem.sql.session.prepared.max-p99`
+  - `sql.mem.sql.session.prepared.max-p99.9`
+  - `sql.mem.sql.session.prepared.max-p99.99`
+  - `sql.mem.sql.session.prepared.max-p99.999`
+
+    [#97590][#97590]
+
+- Active execution information is now shown even when there is a max size limit error. [#97662][#97662]
+- "Retrying" is no longer a status shown in the Jobs page. [#97505][#97505]
+
+<h3 id="v23-1-0-alpha-5-bug-fixes">Bug fixes</h3>
+
+- Transaction uncertainty intervals are now correctly configured for reverse scans, to prevent reverse scans from serving stale reads when clocks in a cluster are skewed. [#97443][#97443]
+- The formatting of uniqueness violation errors now matches the corresponding errors from PostgreSQL. [#96914][#96914]
+- Fixed a bug in `ALTER TABLE ... ADD COLUMN` when the new column name requires quoting due to mixed case or special characters and the statement is not run in an explicit or multi-statement transaction. [#97514][#97514]
```

Comment:
Link to the SQL statement page.
```suggestion
- Previously, when a new column name would require quoting due to mixed-case or special characters, `ALTER TABLE ... ADD COLUMN` would not run in an explicit or multi-statement transaction. This is now fixed. [#97514][#97514]
```

Reply:
I don't think this has to do with the SQL statement page -- this is about an ADD COLUMN that would fail if the column name needed to be quoted.

Reply:
I meant linking to the page for the SQL statement, `alter-table.html#add-column`